.github/workflows/integration_tests.yml

```diff
@@ -492,214 +474,62 @@ jobs:
-        if: env.RUN_AIRGAPPED_TEST != 'true' && env.RUN_CONTINUOUS_DELIVERY_TEST == 'true'
-        timeout-minutes: 7
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run Test_DeliveryAssistant -v
-
-      - name: Test Backup and Restore
-        id: test_backup_restore
-        if: env.BACKUP_RESTORE == 'true'
-        timeout-minutes: 7
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run Test_BackupRestore -v
+      - name: Prepare test run
+        id: prepare_test_run
+        run: |
+          go install gotest.tools/gotestsum@latest
+          cd test/go-tests
+          go get ./...
+          mkdir "${{ env.TEST_REPORT_FOLDER }}"
+          mkdir "${{ env.FINAL_PLATFORM_TEST_REPORT_FOLDER }}"
 
-      - name: Test Resource Service
-        id: test_resource_service
-        if: env.RESOURCE_SERVICE == 'true'
-        timeout-minutes: 7
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run Test_ResourceServiceBasic -v
+      - name: "------- TESTS START -------"
+        run: echo "------- TESTS START -------"
 
-      - name: Test User Managed Delivery
-        id: test_user_managed_delivery
-        if: env.RUN_AIRGAPPED_TEST != 'true' && env.RUN_CONTINUOUS_DELIVERY_TEST == 'true'
-        timeout-minutes: 5
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run CustomUserManagedEndpointsTest -v
+      #######################################################################
+      # TESTS
+      #######################################################################
 
-      - name: Test Continuous Delivery with parallel stages (with sockshop)
-        id: test_continuous_delivery
-        if: env.RUN_AIRGAPPED_TEST != 'true' && (env.RUN_CONTINUOUS_DELIVERY_TEST == 'true') && (env.REMOTE_EXECUTION_PLANE != 'true')
-        timeout-minutes: 45
-        continue-on-error: true
-        env:
-          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
-        run: cd test/go-tests && go test -run ContinuousDelivery -v
+      #######################################################################
+      # COMMON TESTS
+      ##############
 
-      - name: Test Continuous Delivery with parallel stages on Remote Exec Plane (with sockshop)
-        id: test_continuous_delivery_remote_exec
-        if: env.RUN_AIRGAPPED_TEST != 'true' && (env.RUN_CONTINUOUS_DELIVERY_TEST == 'true') && (env.REMOTE_EXECUTION_PLANE == 'true')
+      - name: Integration Tests
+        id: test_aggregated
         timeout-minutes: 45
```

Comment:
I think we need to increase the timeout value since https://github.com/keptn/keptn/runs/4701677577?check_suite_focus=true timed out on the GKE platform tests

Reply:
i already increased the timeout limit to 60 minutes 👍🏼